src/ts/components/playbacktogglebutton.ts

```diff
@@ -33,16 +34,31 @@ export class PlaybackToggleButton extends ToggleButton<ToggleButtonConfig> {
         return;
       }
 
-      if (player.isPlaying()) {
+      // always hide the button on the onPlay Event as the user should get feedback on the press of the play button
+      if (player.isPlaying() || (event && event.type === player.EVENT.ON_PLAY)) {
         this.on();
       } else {
         this.off();
       }
     };
 
+    let playbackWarningHandler = (event: WarningEvent) => {
+      if (event && event.code === 5008) {
```

Comment:
As this handler always receives an event object, there is no need for the first check (`event && ...`).